src/main/java/org/apache/flink/benchmark/ContinuousFileReaderOperatorBenchmark.java

```diff
@@ -19,13 +19,15 @@
 
 import joptsimple.internal.Strings;
 import org.apache.flink.api.common.io.FileInputFormat;
+import org.apache.flink.api.common.restartstrategy.RestartStrategies;
 import org.apache.flink.api.common.typeinfo.TypeInformation;
 import org.apache.flink.configuration.Configuration;
 import org.apache.flink.core.fs.FileInputSplit;
 import org.apache.flink.core.fs.Path;
 import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
 import org.apache.flink.streaming.api.functions.sink.DiscardingSink;
 import org.apache.flink.streaming.api.functions.source.ContinuousFileReaderOperator;
```

Comment:
unused import?

Reply:
The restart strategy was changed to prevent indefinite restarts in the future:
```env.setRestartStrategy(new RestartStrategies.NoRestartStrategyConfiguration());```